### DISCUSSION TITLE
Floor warming - is this possible?

### DISCUSSION THREAD

**Discussion by derJo1** [revk/xESP32-Faikout#508] [General]
Hello, 
I have a Perfera Floor Standing and the Faikin modul works good.
But I missed the Floor Warming funktion, is it possible to integrade this in the faikin module?
Greets 
Jo

**Comment by revk** [comment]
There is a lot of diagnostic work on the S21 protocol being done.

I don't know if they have found "floor warning" yet. If/when they do then it will be easy to add.

**Reply by derJo1** [reply]
Ok, I wait. 😀
Thanks a lot.
